docs/apv.md

```diff
@@ -322,7 +322,8 @@ De overtreding beschreven in lid 1 staat bekend als “spam”
 2. Het is niet toegestaan om te ontsnappen of jezelf uit te laten breken indien er is afgesproken om het roleplayscenario op een later tijdstip voort te zetten, bijvoorbeeld vanwege tijdsgebrek of andere beperkingen.
 3. Het is niet toegestaan om een uit-dienst politiemedewerker te ontvoeren om een ontsnapping van een arrestant uit het cellencomplex te faciliteren of te vergemakkelijken. Deze agenten zijn op dat moment immers niet bezig met de uitvoering van de politietaak.
 4. Het is niet toegestaan om jezelf voor te doen als overheidsmedewerker bij het ontsnappen of helpen ontsnappen van een derde persoon uit het cellencomplex.
-5. Bij overtreding van lid 1, lid 2, lid 3 en/of lid 4 zal er een straf uitgedeeld worden volgens de 1e categorie en zal het roleplayscenario worden teruggedraaid.
+5. Het is verboden om tussen 23:00 en 10:00 uit het cellencomplex te breken.
```

Comment:
moet dit niet in de tijdelijke bepalingen of niet?

Reply:
Nee dit is juist overgenomen vanuit het gevangenis gedeelte omdat het anders niet duidelijk was dat dit ook geldt voor het cellencomplex.

Reply:
Het lijkt mij niet handig om dit toe te voegen aan een ander artikel of het artikel omtrent de gevangenis. Dit artikel is eerder specifiek gemaakt voor het uitbreken uit het cellencomplex en het zou dubbel zijn om het dan ergens anders onder te plaatsen.

Reply:
check okay mee eens